tests/helpers/mobiledoc.js

```diff
@@ -0,0 +1,26 @@
+import PostNodeBuilder from 'content-kit-editor/models/post-node-builder';
+import MobiledocRenderer from 'content-kit-editor/renderers/mobiledoc';
+
+/*
+ * makeMD({post, section, marker, markup} =>
+     post([
+       section('P', [
+         marker('some text', [markup('B')])
+       ])
+     })
+ */
+function makeMD(treeFn) {
```

Comment:
`build` seems fine to me. I'm just railing against abbreviations.

```js
makeMD({post, section, marker, markup} =>
  post([
    section('P', [
      marker('some text', [markup('B')])
    ])
  })
```
is not valid JS. I ported it to something that parses:

```js
makeMD(({post, section, marker, markup}) => {
  return post([
    section('P', [
      marker('some text', [markup('B')])
    ])
  ]);
});
```